### DISCUSSION TITLE
about client tools, cli or GUI, neither avaibale

### DISCUSSION THREAD

**Discussion by yizhouxw** [CeresDB/ceresdb#1403] [Ideas]
there is no client tool in document, what is the recommended client tool for horaedb, either cli or GUI.

some possibile solutions
- offer mysql compabitlbe protocal support, so we can reuse GUI client which have supported mysql.
- offer influxdb/opentsdb compabitle protocal support, so we can reuse GUI client which have supported influxdb.
- offer jdbc driver component, cowork with github.com/oceanbase/odc community.

**Comment by tanruixiang** [comment]
Hi, Thanks for your attention. We already support mysql protocol, you can use any mysql client to connect horaedb.
Horaedb's default mysql port is 3307 you can get it from https://horaedb.apache.org/quick_start.html
